docs/data-masking-function-list.md

```diff
@@ -0,0 +1,894 @@
+
+### Parameters
+
+None.
+
+### Returns
+
+Returns a Canada SIN. The format is `AAA-BBB-CCC` in the `utf8mb4` character set.
+
+The number is verified with the [Luhn Number Checksum]](https://www.dcode.fr/luhn-algorithm) to ensure the number is consistent.
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT gen_rnd_canada_sin();
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------+
+    | gen_rnd_canada_sin()    |
+    +-------------------------+
+    | 506-948-819             |
+    +-------------------------+
+    ```
+
+## gen_rnd_email(name_size, surname_size, domain)
+
+
+
+Generates a random email address. The format of the email is `name.surname@domain`. 
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| name_size | Yes | Specifies the number of characters in the name part. The default is five. | Integer |
+| surname_size | Yes | Specifies the number of characters in the surname part. The default number is seven. | Integer |
+| domain | Yes | Specifies the domain part. The default value is `example.com`. | Integer |
+
+
+### Returns
+
+A generated email address as a string in the same character set as `domain`. If the `domain` value is not specified, then the string is in the `utf8mb4` character set. The `name` and `surname` are random lower-case letters (a - z).
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT gen_rnd_email(name_size=7, surname_size=5, domain=`mydomain.edu`);
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------------------+
+    | gen_rnd_email(4, 5, `mydomain.edu`) |
+    +-------------------------------------+
+    | qwer.asdfg@mydomain.edu             |
+    +-------------------------------------+
+    ```
+
+
+## gen_rnd_iban([country, size])
+
+Generates an Internal Bank Account Number (IBAN).
+
+!!! important
+
+    Generating the IBAN with a valid country code should only be used for testing. The function does not check if the generated value is a legitimate bank account. If you must publish the result, consider using [`mask_iban`](#mask_ibanstr-mask_char) to mix up the result.
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| country | Yes | A two-character country code | String |
+| size | Yes | Number of characters | Integer |
+
+If the `country` is not specified, the default is `ZZ`. The value must be two upper-case characters (A-Z) or an error is returned.
+
+The default length for `size` is 16. The minimum value is 15. The maximum value is 34.
+
+### Returns
+
+The function returns a string based on the `size` value. The string begins with the `country` (two characters) and contains a regular expression in a <country>[[:digit:]]{size} format. The character set is the same as the `country` parameter or if that parameter is not specified, the character set is `utf8mb4`.
```

Comment:
Also, we need to mention that
`IBAN` ckecksum is not calculated, so the generated value may not pass the validation.